composer.json

```diff
@@ -10,27 +10,32 @@
     "issues": "https://github.com/WordPress/two-factor/issues"
   },
   "config": {
+    "sort-packages": true,
     "platform": {
       "php": "5.6.20"
     },
     "allow-plugins": {
-      "roots/wordpress-core-installer": true,
-      "dealerdirect/phpcodesniffer-composer-installer": true
+      "dealerdirect/phpcodesniffer-composer-installer": true,
+      "roots/wordpress-core-installer": true
     }
   },
+  "extra": {
+    "wordpress-install-dir": "wordpress"
```

Comment:
This was needed after updating to `roots/wordpress-full` which has the core themes bundled.